ironfish/src/evm/evm.ts

```diff
@@ -35,6 +39,19 @@ export class IronfishEvm {
     this.blockchainDb = blockchainDb
   }
 
+  async load(): Promise<void> {
+    const stateRoot = await this.blockchainDb.stateManager.getStateRoot()
+
+    if (Buffer.from(stateRoot).equals(NULL_STATE_ROOT)) {
```

Comment:
in the future do you think this should be a db migration or always checked on load? it _should_ only need to run once

Reply:
Feel like the same way we do genesis block makes sense. Maybe both should have been a migration? but want to keep it in the same place as genesis for now